### ISSUE TITLE
Lab 6 Questions

### ISSUE THREAD

**Issue by EngineeringGuy123** [AreteQin/MECH370#19]
For this lab the units are given in microfarads and microhenries, do we have to convert to farads and henries? Also is the resistance 22 ohms?

**Comment by AreteQin** [comment]
Yes

**Comment by EngineeringGuy123** [comment]
Ok thanks. And does the omega or f need to be plotted with a log scale along the x-axis? I read the manual, but it's not clear to me.

**Comment by AreteQin** [comment]
Yes. Because in the Bode plot, both the x and y axes are in log scale. It is better this way to compare with simulations.

**Comment by EngineeringGuy123** [comment]
![Screenshot 2023-08-15 223515](https://github.com/AreteQin/MECH370/assets/140431296/40495450-5c17-4ba5-aaf0-a79c9e9b0ab6)

what is the name of that black rectangle?

**Comment by AreteQin** [comment]
Mux block:

https://www.mathworks.com/help/simulink/slref/mux.html
